src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Resources.resx

```diff
@@ -118,73 +118,73 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="AssemblyNameDoesNotExist" xml:space="preserve">
-    <value>Assembly with name '{0}' does not exist at {1}.</value>
+    <value>'{0}' assembly does not exist at {1}.</value>
   </data>
   <data name="AssemblyNameDoesNotMatch" xml:space="preserve">
-    <value>{2} assembly name '{0}' does not match with {3} assembly name '{1}'.</value>
+    <value>'{0}' assembly name does not match '{1}'.</value>
   </data>
   <data name="AssemblyPublicKeyTokenDoesNotMatch" xml:space="preserve">
-    <value>{2} assembly public key token '{0}' does not match with {3} '{1}'.</value>
+    <value>'{0}' assembly public key token does not match '{1}'.</value>
   </data>
   <data name="AssemblySearchDirectoryDoesNotExist" xml:space="preserve">
-    <value>Provided assembly search directory '{0}' does not exist.</value>
+    <value>Provided search directory '{0}' does not exist.</value>
   </data>
   <data name="CannotAddAbstractMember" xml:space="preserve">
-    <value>Cannot add abstract member '{0}' to {1} because it does not exist on {2}</value>
+    <value>Cannot add abstract member '{0}' to {1}.</value>
   </data>
   <data name="AssemblyCultureDoesNotMatch" xml:space="preserve">
-    <value>{2} assembly culture '{0}' does not match with {3} assembly culture '{1}'.</value>
+    <value>'{0}' assembly culture does not match '{1}'.</value>
```

Comment:
this and the one below lost information.

Reply:
@Thiago-Bastani did you check the result before putting up the PR? I think it's interesting to try using ChatGPT to help, but obviously humans have to check the results before making a PR😄

Reply:
I'll take some time to do it myself... then I'll request reviws again.
as soon as I can :)